.github/workflows/deploy.yml

```
@@ -69,8 +74,7 @@ jobs:
     - name: Post to Splunk SOAR API
       if: github.event_name == 'release' || github.event_name == 'pull_request'
```

I think this should be:

```suggestion
      if: (github.event_name == 'release' || github.event_name == 'pull_request') && github.event.repository.name != 'splunk-soar-template'
```